app/lib/proxy_api/available_proxy.rb

```diff
@@ -13,14 +13,26 @@ class AvailableProxy
 
     def initialize(args)
       @features = ::ProxyAPI::Features.new(args).features
+      @versions = ::ProxyAPI::Version.new(args).proxy_versions['modules']
     end
 
     def available?
       begin
-        return true if @features.include?('openscap')
+        return true if (@features.include?('openscap') && minimum_version)
       rescue *HTTP_ERRORS
         return false
       end
     end
+
+    private
+
+    def openscap_proxy_version
+      @versions['openscap'] if @versions['openscap']
```

Comment:
I think it can be simplified to `@versions['openscap']` unless you want to convert false -> nil, but I guess `@versions['openscap']` should never be false